package.json

```diff
@@ -7,7 +7,7 @@
     "browserify-shim": "^3.8.8",
     "holderjs": "^2.8.0",
     "jquery": "^2.1.4",
-    "kss-bootstrap": "~0.4.3",
+    "kss-bootstrap": "kalamuna/kss-bootstrap#3fb55ed3dd1c35dd118db6866cf2660f792a097d",
```

Comment:
@RobLoach what are the implications here?
Should we not roll a 0.4.4 version of kss-boostrap instead?